<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Punto_Fijo</class>
 <widget class="QWidget" name="Punto_Fijo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table_resoult">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_solve">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>611</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="bt_funcion">
      <property name="text">
       <string>Ingresar Valores</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>Tolerancia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Error abs</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Error relativo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_solve">
      <property name="text">
       <string>Solucionar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ingregar funcion de esta forma :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="plainText">
    <string>numeros ([2.23e-308 hasta 1.79e308) , parentesis() , operadores(+,-,*,/) , seno(&quot; sen(valor) &quot;) ,  coseno(&quot; cos(valor) &quot;) , potencia(&quot;base ^ elevacion&quot;) , raiz cuadrada(&quot; sqrt(valor) &quot;) , exponencial(&quot; e(valor) &quot;)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
